pals/mysterious-halfling.md

```diff
@@ -0,0 +1,65 @@
+
+# ??? It Is A Mystery
+
+> ??? This Is Also A Mystery
+
+#### Species, Class, Level:
+Halfing Rogue, Level 1
+
+#### Languages known:
+Common, +1 other
```

Comment:
You can pick from:
* Dwarven (what dwarves speak)
* Elven (what elves & fey creatures speak)
* Draconic (dragons, kobolds, reptile people etc)
* Giant (giants, orcs, hobgoblins)
* Goblin (goblins, bugbears, and almost every orc & hobgoblin will also know this)
* Deep Speech (mostly-evil things from the Underdark)
* Primordial (elementals and things from Beyond Time)
Whatever you pick will probably hint at a backstory, if/when we get that far.